src/main/java/plugin/TuebIlias.java

```diff
@@ -40,17 +41,17 @@ public LoginStatus login(String username, String password) {
 
 			Document doc = Jsoup.parse(html);
 			Element form = doc.select("form[action*=idp").first();
-			Element input_tag = form.select("input[name='lt']").first();
-			String ltKey = input_tag.attr("value");
-			
+
+			// Updated 06.02.2020 to fit the new login process
```

Comment:
no need to put a comment with dates for updates - git is your friend :)

Reply:
I looked at the GPL2 and was not sure if I had to mark the changes^^